### PR TITLE
fix(identical-poll-options-behaviour-web)

### PR DESCRIPTION
**Pull request by HazemAbdo**
Fixed Identical poll options behavior https://github.com/jitsi/jitsi-meet/issues/14540
- I made a check on the list of options and if I found identical options I disabled the send poll button
- Highlight input field with value that violates the rule and a label that says "Options must be unique" 
- Adding all supported languages translation for this label
- I used https://github.com/ssut/py-googletrans for automating this translation process

image

image

image

### CODE REVIEW

**Comment by HazemAbdo** [comment]
I have already sign the CLA:

image

[8x8_Individual_CLA_rev.20181129.docx.pdf](https://github.com/jitsi/jitsi-meet/files/14751629/8x8_Individual_CLA_rev.20181129.docx.pdf)

**Comment by HazemAbdo** [comment]
I have already signed the CLA

File: react/features/polls/components/AbstractPollCreate.tsx [hunk 1]
```diff
@@ -37,6 +37,18 @@ export type AbstractProps = InputProps & {
  * @param {React.AbstractComponent} Component - The concrete component.
  * @returns {React.AbstractComponent}
  */
+
+const haveIdenticalAnswers = (currentAnswers: Array<string>): boolean => {
```
[the remark this answers is not in the document]
Comment (HazemAbdo) [reply react/features/polls/components/AbstractPollCreate.tsx:41]: I will do it right now.It was a double check to make sure I don't mutate the original array but as the filter function and set constructor return don't mutate it I should remove it.

File: react/features/polls/components/web/PollCreate.tsx [hunk 2]
```diff
@@ -201,13 +201,20 @@ const PollCreate = ({
...
+                            prevAnswer === answer && prevAnswer !== '' && answer !== '');
+
+                    return (<li
                         className = { classes.answer }
                         key = { i }>
                         <Input
+                            error = { isIdenticalAnswer }
                             id = { `polls-answer-input-${i}` }
-                            label = { t('polls.create.pollOption', { index: i + 1 }) }
+                            label = { (isIdenticalAnswer ? t('polls.errors.notUniqueOption', { index: i + 1 })
```
[the remark this answers is not in the document]
Comment (HazemAbdo) [reply react/features/polls/components/web/PollCreate.tsx:216]: sure I will do it also with the previous comment.Thank you for your feedback

[the remark this answers is not in the document]
Comment (HazemAbdo) [reply react/features/polls/components/web/PollCreate.tsx]: - I moved the function to /jitsi-meet/react/features/polls/functions.ts
- Get rid of unused JSON.parse 
- Used bottomLabel for showing error message

Untitled
Untitled

**Comment by mihhu** [comment]
Could you please move the translation key to have the JSONs keys ordered alphabetically, in order to fix the lint check?

**Comment by HazemAbdo** [comment]
@mihhu  I run the command `npm run lang-sort` as you mentioned I didn't catch this lint problem locally as when I run lint-fix,lint:lang locally it didn't show this problem but I just made a push with sorted JSON keys.Thank you for all your pieces of advice

**Review by mihhu — APPROVED** [review]

**Review by rpurdel — APPROVED** [review]

**Pull request merged**